### PR TITLE
Remove links to orcharhino docs

### PR DESCRIPTION
**Pull request by maximiliankolb**
Cherry-pick into:

* [x] Foreman 3.2
* [x] Foreman 3.1

### CODE REVIEW

File: guides/common/modules/con_configuring-the-discovery-service.adoc [hunk 1]
```diff
@@ -56,11 +56,8 @@ endif::[]
 
 The `tfm-rubygem-smart_proxy_discovery` package configures {SmartProxyServer}, such as the integrated {SmartProxy} of {ProjectServer}, to act as a proxy for the Discovery service.
 
-ifdef::foreman-el,katello[]
+ifdef::foreman-el,katello,orcharhino[]
```
Comment (ekohl) [inline guides/common/modules/con_configuring-the-discovery-service.adoc:59]: I actually wonder if this shouldn't be
```suggestion
infdef::satellite[]
```

**Review by ekohl — COMMENTED** [review]
Any reason this shouldn't be picked to 3.2? I always think it's odd when something is in master and x.y but not x.(y+1).

File: guides/common/modules/con_configuring-the-discovery-service.adoc (hunk 1 above)
Comment (maximiliankolb) [reply guides/common/modules/con_configuring-the-discovery-service.adoc]: Yes, makes sense. Applied.

**Comment by maximiliankolb** [comment]
Rebased to master and applied your suggestion. I did not select 3.2 because I'm lazy. But yes, I can cherry-pick there as well.

**Review by ekohl — APPROVED** [review]

**Comment by maximiliankolb** [comment]
Merged and cherry-picked to 3.1 and 3.2: https://github.com/theforeman/foreman-documentation/commit/6935719b091e196da1eeceb6cba3590e430873b8 and https://github.com/theforeman/foreman-documentation/commit/49fd006dbd52c750fd5940455bf8f5825db435ea ✅

**Pull request merged**